actions/arteria_delivery_service.py

```diff
@@ -267,13 +267,10 @@ def run(self, action, delivery_base_api_url, irma_api_key, sleep_time, **kwargs)
                                                        staging_id=kwargs['staging_id'],
                                                        md5sum_file=kwargs.get('md5sum_file'),
                                                        skip_mover=skip_mover)
-
-            # Wait a short time and then try to update the status, just to
-            # make sure we fail in this step if we get a failed status early.
-            time.sleep(5)
-            service.update_delivery_status(project_and_delivery_id, skip_mover=skip_mover)
-
-            exit_flag = project_and_delivery_id.is_in_progress(skip_mover)
+            project_and_delivery_id.set_status(
+                project_and_delivery_id.mover_processing_delivery,
+                skip_mover=skip_mover)
+            exit_flag = True
```

Comment:
Perhaps we should rather use something like this?
```
if project_and_delivery_id:
    exit_flag = True
else:
    exit_flag = False
```
So that if there is a problem with the first posting for some reason which does not raise an exception, this will not make the action run return 0.